README.md

```diff
@@ -13,7 +13,7 @@ wolfSSL. The following is the simplest configuration of wolfSSL to
 enable wolfSSH.
 
     $ cd wolfssl
-    $ ./configure [OPTIONS] --enable-ssh
+    $ ./configure [OPTIONS] --enable-wolfssh
```

Comment:
This option was only added recently in https://github.com/wolfSSL/wolfssl/pull/5100 . For now please leave the `--enable-ssh`.